branchLayoutApi/src/main/java/com/virtuslab/branchlayout/api/BranchLayoutException.java

```diff
@@ -1,15 +1,16 @@
 package com.virtuslab.branchlayout.api;
 
 import io.vavr.control.Option;
+import org.checkerframework.checker.index.qual.NonNegative;
 import org.checkerframework.checker.index.qual.Positive;
 import org.checkerframework.checker.nullness.qual.Nullable;
 
 public class BranchLayoutException extends Exception {
   @Nullable
-  @Positive
+  @NonNegative
```

Comment:
Hmm I think `@Positive` was correct here, the lines should be numbered from 1, not 0